app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by their name. This approach assumes that names are unique, which might not be a valid assumption in all cases. Consider using a unique identifier for each person if possible.